modules/miraheze.py

```diff
@@ -43,10 +43,23 @@ def miraheze_gethelp(bot, trigger):
         bot.reply(trigger.nick + ', needs help. Pinging Reception123, Zppix, '
                 'PuppyKun, Voidwalker, RhinosF1.')
     else:
-        bot.reply('If you need Miraheze releated help, please join #miraheze')
+         bot.reply('If you need Miraheze releated help, please join #miraheze')
 
 
 @commands('discord')
 def miraheze_discord(bot, trigger):
     """Displays discord information for Miraheze."""
     bot.reply('You can join discord by going to, https://discord.is/miraheze!')
+    
+@rule('.*[w-wW-W]hat (even is [m-mM-M]iraheze|is [m-mM-M]iraheze|does [m-mM-M]iraheze do).*')
+def miraheze(bot, trigger):
+    """
+    Miraheze about command.
+
+    This command will tell you about Miraheze and where to learn more.
+    """
+    if trigger.sender == '#miraheze':
+        bot.reply(MIRAHEZE_ABOUT_MIRAHEZE_CHANNEL)
+    else:
+        bot.reply(MIRAHEZE_ABOUT_OTHER_CHANNELS)
```

Comment:
This is a complete duplicate.

Move this to a miraheze(bot) method and have the rule & command call it